op-guide/migration.md

```diff
@@ -201,6 +198,29 @@ mysql> select * from t2;
 +----+------+
 ```
 
+### 3. Best practice
+In order to migrate data quickly (especially huge amount of data), you can refer to the following recommendations.
```

Comment:
In order to migrate data quickly (especially huge amount of data)
->
To migrate data quickly, especially for huge amount of data,